pkgs/by-name/in/inter-nerdfont/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  fontforge,
+  nerd-font-patcher,
+  stdenvNoCC,
+  fetchzip,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "inter-nerdfont";
+  version = "4.0";
+
+  src = fetchzip {
+    url = "https://github.com/rsms/inter/releases/download/v${finalAttrs.version}/Inter-${finalAttrs.version}.zip";
+    stripRoot = false;
+    hash = "sha256-hFK7xFJt69n+98+juWgMvt+zeB9nDkc8nsR8vohrFIc=";
+  };
+
+  buildPhase = ''
+    runHook preBuild
+    nerd-font-patcher Inter.ttc
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm444 'Inter Nerd Font.ttc' $out/share/fonts/truetype/InterNerdFont.ttc
+    cp *.ttf $out/share/fonts/truetype
+    runHook postInstall
+  '';
+
+  buildInputs = [
```

Comment:
I'm changing this to `nativeBuildInputs` as these are binaries not meant to end up in the runtime closure.